TLM/TLM/Resources/lang_fr.txt

```diff
@@ -57,36 +57,36 @@ Enable_Advanced_Vehicle_AI Activer l'IA avancée des véhicules
 Vehicles_may_enter_blocked_junctions Les véhicules peuvent entrer dans les intersections bloquées
 All_vehicles_may_ignore_lane_arrows Tous les véhicules peuvent ignorer les directions des voies
 Busses_may_ignore_lane_arrows Les bus peuvent ignorer les directions des voies
-Reckless_driving Conduite téméraire (fonctionnalité Bêta)
-TMPE_Title Traffic Manager: Edition Président
+Reckless_driving Conduite téméraire
+TMPE_Title Gestionnaire de trafic : Édition Président
 Settings_are_defined_for_each_savegame_separately Paramètres définis pour chaque sauvegarde séparément
 Simulation_accuracy Précision de la simulation (une plus haute précision réduit les performances)
 Enable_highway_specific_lane_merging/splitting_rules Activer les règles spécifiques de divergence/convergence sur les autoroutes
-Drivers_want_to_change_lanes_(only_applied_if_Advanced_AI_is_enabled): Les conducteurs veulent changer de voies: (seulement si l'IA avancée est activée)
+Drivers_want_to_change_lanes_(only_applied_if_Advanced_AI_is_enabled): Les conducteurs veulent changer de voies (seulement si l'IA avancée est activée) : 
 Maintenance Maintenance
 Very_often Très souvent
 Often Souvent
 Sometimes Parfois
 Rarely Rarement
 Very_rarely Très rarement
 Only_if_necessary Seulement si nécessaire
-Nodes_and_segments Nœuds et segments
+Nodes_and_segments Intersections et segments
 Lanes Voies
 Path_Of_Evil_(10_%) Chemin du Mal (10 %)
-Rush_Hour_(5_%) Heures de Pointe (5 %)
+Rush_Hour_(5_%) Heure de Pointe (5 %)
 Minor_Complaints_(2_%) Plaintes mineures (2 %)
 Holy_City_(0_%) Ville Sainte (0 %)
 Forget_toggled_traffic_lights Oublier les feux tricolores retirés
-Road_is_already_in_a_group! Cette route est déjà dans un groupe !
-All_selected_roads_must_be_of_the_same_type! Toutes les routes sélectionnées doivent être du même type!
+Road_is_already_in_a_group! Cette route est déjà dans un groupe !
+All_selected_roads_must_be_of_the_same_type! Toutes les routes sélectionnées doivent être du même type !
 Create_group Créer un groupe
 Delete_group Supprimer un groupe
 Add_zoning Ajouter le zonage
 Remove_zoning Retirer le zonage
 Lane_Arrow_Changer_Disabled_Highway Le modificateur de directions pour cette voie est désactivé car vous avez activé le système de règles pour autoroutes.
 Add_junction_to_timed_light Ajouter une intersection à ce feu tricolore
 Remove_junction_from_timed_light Retirer une intersection de ce feu tricolore
-Select_junction Sélectionnez une intersection
+Select_junction Sélectionner une intersection
```

Comment:
I'd like to ask for a bit of context here, as both forms may be correct, and I might have wrongly modified that !

So `Sélectionnez` with a `z` is the imperative form ; with an `r` is the infinitive.

Which one would fit in game ?

Reply:
Whichever seems more correct in French should be used.

In English, I'd probably go with the imperative form.